sdk/storage/Azure.Storage.Queues/src/Models/DequeuedMessageItem.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Queues.Models
+{
+    internal partial class DequeuedMessageItem
```

Comment:
what is this for? (I can see more empty partial classes).

Reply:
This is how you tell the generator to make this type internal.  This is by design.